src/v2/Apps/Fair/Components/FairHeader/FairTiming.tsx

```diff
@@ -12,25 +13,31 @@ interface Props {
 const FairTiming: React.FC<Props> = ({
   fair: { exhibitionPeriod, startAt, endAt },
 }) => {
+  const currentTime = useCurrentTime({ syncWithServer: true })
+
+  const renderEventTiming = () =>
+    startAt &&
+    endAt && (
+      <EventTiming currentTime={currentTime} startAt={startAt} endAt={endAt} />
+    )
+
   return (
     <>
-      <Text variant="xl" color="black60" mb={1}>
-        {exhibitionPeriod}
-      </Text>
+      {/* Desktop Fair Timing */}
+      <Media greaterThan="xs">
```

Comment:
**suggestion**: Similar comment to the one above to explore using [responsive props](https://palette.artsy.net/guides/responsive/) instead of the `Media` component.